Configuration/StandardSequences/python/Services_cff.py

```diff
@@ -13,9 +13,15 @@
 def _addProcessAccelerators(process):
     process.load("Configuration.StandardSequences.Accelerators_cff")
 
+def _addProcessAcceleratorsAlpaka(process):
+    process.load("HeterogeneousCore.AlpakaCore.ProcessAcceleratorAlpaka_cfi")
```

Comment:
I would prefer the loading of `ProcessAcceleratorAlpaka_cfi` to be in `Accelerators_cff`. I also think that, given the state of the GPU matters in the offline side, the import there could be unconditional.

Reply:
Yes.

I think adding the `alpaka` modifier as one of the ways to call `makeProcessModifier(_addProcessAccelerators)` below is fine (or, I'm not against).